Gemfile

```diff
@@ -9,17 +9,8 @@ gem 'omnibus', git: 'https://github.com/rapid7/omnibus', branch: 'r7_working'
 # gem 'omnibus-software', github: 'opscode/omnibus-software'
 
 # This development group is installed by default when you run `bundle install`,
-# but if you are using Omnibus in a CI-based infrastructure, you do not need
-# the Test Kitchen-based build lab. You can skip these unnecessary dependencies
-# by running `bundle install --without development` to speed up build times.
+# You can skip these unnecessary dependencies by running
+# `bundle install --without development` to speed up build times.
 group :development do
-  # Use Berkshelf for resolving cookbook dependencies
-  gem 'berkshelf', '~> 3.0'
-
-  # Use Test Kitchen with Vagrant for converging the build environment
-  gem 'test-kitchen'
-  gem 'kitchen-vagrant'
-  gem 'vagrant-wrapper'
-
   gem 'nio4r', '~> 1.2.1'
```

Comment:
I have a feeling this might not be needed now - or it's maybe in the wrong group. But couldn't I find a corresponding PR without any context, seems to be a direct commit to master related to ruby version issues -https://github.com/rapid7/metasploit-omnibus/commit/e3932234d97adb3178cbfa967f33dd047da75ad9

Reply:
From what I can tell it's a transitive dependency of berkshelf, via celluloid

Reply:
Likely, yeah let's kill it too if unneeded.